feedback-module/src/index.js

```diff
@@ -4,38 +4,51 @@ import "@trussworks/react-uswds/lib/index.css";
 import "./i18n";
 import "./styles/index.scss";
 import App from "./App";
+import GA4React from "ga-4-react";
 
 const WidgetDivs = document.querySelectorAll("#feedback-widget");
 
-const renderApp = (Div, lang, pagetitle, endpoint) => {
+const renderApp = (Div, lang, pagetitle, endpoint, gaid) => {
   Div.attributes.lang && (lang = Div.attributes.lang.value);
   Div.attributes.pagetitle && (pagetitle = Div.attributes.pagetitle.value);
   Div.attributes.endpoint && (endpoint = Div.attributes.endpoint.value);
-  ReactDOM.render(
-    <React.StrictMode>
-      <Suspense fallback="... is loading">
-        <App
-          domElement={Div}
-          lang={lang}
-          pagetitle={pagetitle}
-          endpoint={endpoint}
-        />
-      </Suspense>
-    </React.StrictMode>,
-    Div
-  );
+  Div.attributes.gaid && (gaid = Div.attributes.gaid.value);
+
+  const ga4react = new GA4React(gaid);
```

Comment:
Do we need to create a new GA4React object every time the app is re-rendered (i.e. could we put this line outside of the renderApp function)? Not too familiar with how GA4React works, so I'm just wondering.

Reply:
Right now we are getting the gaid attribute within the render app which is why I put this here. I guess we could make the assumption that the user won't change their gaid so we wouldn't need to observe for it?

Reply:
Hmmm that's a good point— I guess we can keep it how it is just in case?